src/layout/umap.c

```diff
@@ -1027,6 +1026,8 @@ igraph_error_t igraph_layout_umap(const igraph_t *graph,
  *
  * \return Error code.
  *
+ * Added in version 0.10. This function is experimental and the API is not stable.
```

Comment:
I mean literally add `\experimental`, which will expand to a big yellow warning box :-) I can also do this, unless someone else gets to it first.